xbmc/filesystem/Directory.h

```diff
@@ -79,6 +79,6 @@ class CDirectory
   /*! \brief Filter files that act like directories from the list, replacing them with their directory counterparts
    \param items The item list to filter
    \param mask  The mask to apply when filtering files */
```

Comment:
Could you please add the documentation for the new parameter here? It's now incomplete